django3_codemods/config_tools/tox_modernizer.py

```diff
@@ -0,0 +1,102 @@
+import sys
+import re
+from configparser import ConfigParser, NoSectionError
+
+TOX_SECTION = "tox"
+ENVLIST = "envlist"
+TEST_ENV_SECTION = "testenv"
+TEST_ENV_DEPS = "deps"
+PYTHON_SUBSTITUTE = "py{35,38}"
+DJANGO_SUBSTITUTE = "django{22,30}"
+
+DJANGO_22_DEPENDENCY = "django22: Django>=2.2,<2.3\n"
+DJANGO_30_DEPENDENCY = "django30: Django>=3.0,<3.1\n"
+NEW_DJANGO_DEPENDENCIES = DJANGO_22_DEPENDENCY + DJANGO_30_DEPENDENCY
+
+SECTIONS = [TOX_SECTION, TEST_ENV_SECTION]
+
+PYTHON_PATTERN = "(py{.*?}-|py[0-9]+,|py[0-9]+-)"
+
+DJANGO_PATTERN = "(django[0-9]+,|django[0-9]+\n|django{.*}\n|django{.*?}|django[0-9]+-|django{.*}-)"
```

Comment:
I'm not taking the time to validate that this does exactly the right thing; if we have issues with it, we can always update the script accordingly later.